### PR TITLE
use displayNames in legends

### PR DESCRIPTION
**Pull request by JowieXiang**
Added a `config.js` for storing all service objects. It was necessary to take the objects out of `map/scripts.js` because:
1. `map/legend.js` has to be added before `map/script.js` in the index page 
2. for `map/legend.js` to use the service objects, `map/config.js` has to be added before `map/legend.js` in the index page  

If this pull request is approved, I will update the `bhubaneshwar` and `latacunga` branch accordingly.

### CODE REVIEW

File: webapp/views/map/script.js [hunk 1]
```diff
@@ -108,19 +32,20 @@ const baseLayers = {
...
-    "Waterways": waterways,
-    "Roads": roads,
-    "Buildings": buildings,
-    "Basemap boundary": basemapBbox,
-    "Current selection": selection
+    [services.waterways.displayName]: createWms(services.waterways),
+    [services.roads.displayName]: createWms(services.roads),
+    [services.buildings.displayName]: createWms(services.buildings),
+    [services.basemapBbox.displayName]: createWms(services.basemapBbox),
+    [services.selection.displayName]: createWms(services.selection)
```
Comment (janbehrens) [inline webapp/views/map/script.js:42]: This should be a one-liner!

**Comment by JowieXiang** [comment]
Moved grouping logic into config

File: webapp/views/map/script.js (hunk 1 above)
Comment (JowieXiang) [reply webapp/views/map/script.js]: Moved grouping logic into config in the newest commit

File: webapp/views/map/script.js [hunk 2]
```diff
@@ -33,21 +33,15 @@ const baseLayers = {
...
-  "Time map": {
-    [services.fromPoints.displayName]: createWms(services.fromPoints),
-    [services.viaPoints.displayName]: createWms(services.viaPoints),
-    [services.strickenArea.displayName]: createWms(services.strickenArea),
-    [services.timeMap.displayName]: createWms(services.timeMap)
-  }
-};
+// Configure the layer switcher
+const groupedOverlays = {}
+const groups = [...new Set(services.map(ser=>ser.group))]
```
Comment (janbehrens) [inline webapp/views/map/script.js:38]: How can I control the ordering of the groups?

Comment (JowieXiang) [reply webapp/views/map/script.js:38]: By putting them in order in the config.js.

**Review by janbehrens — APPROVED** [review]

**Pull request merged**